python_exception_quiz
